main_tests/fmcdaq3/production.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+SCRIPT_DIR="$(readlink -f $(dirname $0))"
+source "$SCRIPT_DIR/../lib/utils.sh"
+
+MODE="$1"
+case $MODE in
+	"FMCDAQ3 Test")
+		"$SCRIPT_DIR/rf_test.sh";
```

Comment:
why the quotation marks?

Reply:
I added the quotes to allow for spaces in the file path. Not absolutely necessary for the prodtest images - this is likely a harmless leftover from one of my test runs. Should I remove it the quotes consistency's sake?

Reply:
yes